magnum-zio/src/test/scala/com/augustnagro/magnum/magzio/PgZioTests.scala

```diff
@@ -0,0 +1,48 @@
+package com.augustnagro.magnum.magzio
+
+import com.augustnagro.magnum.*
+import com.dimafeng.testcontainers.PostgreSQLContainer
+import com.dimafeng.testcontainers.munit.fixtures.TestContainersFixtures
+import munit.{AnyFixture, FunSuite, Location}
+import org.postgresql.ds.PGSimpleDataSource
+import org.testcontainers.utility.DockerImageName
+
+import java.nio.file.{Files, Path}
+import scala.util.Using
+import scala.util.Using.Manager
+
+class PgZioTests extends FunSuite, TestContainersFixtures:
+
+  immutableRepoZioTests(this, PostgresDbType, xa)
```

Comment:
I only re-implemented the `immutableRepoTests`. Not sure if there's any value in re-implementing more of them 🤔